packages/ui/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FocusEventHandler, MouseEventHandler, ReactElement } from 'react';
+import { forwardRef, IconButton as ChakraIconButton, IconButtonProps as ChakraIconButtonProps } from '@chakra-ui/react';
+
+type IconButtonSize = 'sm' | 'md' | 'lg' | 'xs';
+
+type IconButtonColor = 'default' | 'brand' | 'danger';
+
+type IconButtonVariant = 'link' | 'outline' | 'solid' | 'ghost';
+
+export interface IconButtonProps {
+  icon: ReactElement;
+  /**
+   * Color variant of the icon button
+   */
+  color?: IconButtonColor;
+  /**
+   * Determines button's width, height and icon's size.
+   */
+  size?: IconButtonSize;
+  /**
+   * Style variant of the icon button
+   */
+  variant?: IconButtonVariant;
+  onClick?: MouseEventHandler;
+  onFocus?: FocusEventHandler;
+  onBlur?: FocusEventHandler;
+  'aria-label'?: string;
+}
+
+const getColorScheme = (color: IconButtonColor): ChakraIconButtonProps['colorScheme'] => {
+  if (color === 'danger') {
+    return 'red';
+  }
+
+  if (color === 'brand') {
+    return 'brand';
+  }
+
+  return 'gray';
+};
+
+const getFontSize = (size: IconButtonSize): ChakraIconButtonProps['fontSize'] => {
```

Comment:
I think there is a cleaner and shorter way of doing this. You can create an object like follows:
```
const fontSizes = {
    xs: 13,
    sm: 16,
    md: 20,
    lg: 23
} 
```
Then you can use such object as `fontSize={fontSizes[size]}`. This applies to `getColorScheme` as well.

Other cases here are the wrong sizes of icons. To match mockups it should be:
xs: 16
sm: 20,
md: 24,
lg: 28